src/Common/DuplicateSubmitButton.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace ipl\Web\Common;
+
+use ipl\Html\Attribute;
+use ipl\Html\Contract\FormSubmitElement;
+use ipl\Html\FormElement\SubmitElement;
+
+trait DuplicateSubmitButton
+{
+    /**
+     * Creates a Duplicate Submit Element with an added `class` attribute
+     *
+     * @param FormSubmitElement $originalSubmitButton
+     *
+     * @return SubmitElement
+     */
+    protected function duplicateSubmitButton(FormSubmitElement $originalSubmitButton): SubmitElement
+    {
+        $attr = clone $originalSubmitButton->getAttributes();
```

Comment:
Aren't we cloning the element because of a possible wrapper or because the cloning PR hasn't been merged yet?

Reply:
Primarily because the cloning PR isn't merged yet. But I also haven't tested if there could be an unwanted wrapper if I would clone the whole element.